wifiphisher.py

```diff
@@ -869,6 +869,14 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n\033[91mUnable to install hostapd from the source!\033[0m\n' +
+            "Please follow the link below to configure your sources.list\n" +
+            "\033[94mhttp://docs.kali.org/general-use/kali-linux-sources-list-repositories\033[0m\n" +
```

Comment:
We have some <a href="https://github.com/HassenPy/wifiphisher/blob/patch-4/wifiphisher.py#L32-L41">constants for the coloring</a>. You should make use of them.

Also, try to follow our convention for error messages.We usually add a `'['+R+'-'+W+']'` before the actual message. Check <a href="https://github.com/HassenPy/wifiphisher/blob/patch-4/wifiphisher.py#L870">here</a> for example.